### DISCUSSION TITLE
About the cpu usage calculation.

### DISCUSSION THREAD

**Discussion by weiqiang-w** [apache/skywalking#7743] [Q&A]
what's the difference of  between 
cpuCost * 1.0d / ((now - lastSampleTimeNs) * cpuCoreNum) * 100
and
OperatingSystemMXBean.getProcessCpuLoad();

the first one  is provide by skywalking .the second one is provide by sun management.

**Comment by wu-sheng** [comment]
Explaing JVM codes are not SkyWalking responsibility

**Reply by weiqiang-w** [reply]
okok. still  thanks for answering.
